doc/user/content/cloud/quickstart.md

```diff
@@ -34,17 +34,21 @@ By default, you can create up to two deployments. If you're interested in more,
 
 ## Connect to a real-time stream and a create materialized view
 
-For this example, we'll walk you through connecting to a [PubNub stream](https://www.pubnub.com/developers/realtime-data-streams/) as a data source. Note that PubNub demo streams should only be used for testing, since they do not meet the consistency and durability requirements necessary for Materialize to guarantee correctness over time.
+For this example, we'll walk you through connecting to a [PubNub stream](https://www.pubnub.com/developers/realtime-data-streams/) as a data source. Note that PubNub demo streams should only be used for testing, since they are [volatile sources](/overview/volatility) that do not meet the consistency and durability requirements necessary for Materialize to guarantee correctness over time.
 
-1. From your shell, create a source (connect to the PubNub market orders stream):
+1. If you don't already have a PubNub account, [create one](https://dashboard.pubnub.com/signup) to obtain a subscribe key. (This is free.)
+
+2. From your shell, create a source (connect to the PubNub market orders stream):
 
     ```sql
     CREATE SOURCE market_orders_raw FROM PUBNUB
-      SUBSCRIBE KEY 'sub-c-4377ab04-f100-11e3-bffd-02ee2ddab7fe'
```

Comment:
Is it necessary to force users to get their own subscribe key?  This key is publicly available here: https://www.pubnub.com/developers/realtime-data-streams/financial-securities-market-orders/

It's a nice user experience to be able to copy-paste the snippet and not have to sign up for a new account

Reply:
The one in the Developer doc is Petros' personal key, and we can use that or mine. They're probably fine for a small number of users, but they're rate-limited and may run into problems when we have more.

Reply:
I'm pretty sure it's not Petros' personal key!  It's listed on PubNub's website for everyone to use